packages/express-session/src/index.ts

```diff
@@ -0,0 +1,98 @@
+import { AccountsServer } from '@accounts/server';
+import { Tokens, User } from '@accounts/types';
+import { Request, Response, NextFunction } from 'express';
+import * as requestIp from 'request-ip';
+import 'express-session';
+import { getUserAgent } from './utils/get-user-agent';
+
+export interface AccountsSessionOptions {
+  user: {
+    name: string;
+    resolve: (tokens: Tokens) => User | Promise<User>;
```

Comment:
Was wondering for this function would be great to have a default behavior? same as the current express server.